how-to/releaseduty/merge-duty/merge_new_esr_branch.rst

```diff
@@ -74,7 +74,7 @@ Task list and known dependencies
     mozilla-esrXX (the goal is to get a first set of tasks, so pick a
     non-DONTBUILD push :) )
 
-11. After 6, add esrXX to cross-channel configuration
+11. After 6, add esrXX to [l10n automation](https://github.com/mozilla-l10n/firefox-l10n-source/blob/2e3f7c95af14ed6e17501195e738e28c8282fc4b/.github/update-config.json) configuration
```

Comment:
should we use a non-permalink here?

Reply:
I used a perma link in case the file path ever changes to avoid having a broken link. I don't mind either way

Reply:
My thinking was if it's a broken link then it's a good signal that the doc is out of date?

Reply:
Also re-reading helped. This is rst, not markdown, this is not how you make a link 😂